tests/pgkit_test.go

```diff
@@ -127,6 +127,23 @@ func TestSugarInsertAndSelectRecords(t *testing.T) {
 	assert.Equal(t, "joe", accounts[0].Name)
 }
 
+func TestGetOneInTransaction(t *testing.T) {
+	ctx := context.Background()
+	q2 := DB.SQL.Select("*").From("accounts")
+
+	var account Account
+	tx, err := DB.Conn.BeginTx(ctx, pgx.TxOptions{})
+	assert.NoError(t, err)
+
+	defer tx.Rollback(ctx)
```

Comment:
This feels wrong - you are committing 5 lines below this, so you'd be rolling back once outside of tx

Reply:
You are right :pray: 

https://github.com/goware/pgkit/pull/19

Reply:
nice catch @c2h5oh :) thanks!